### PR TITLE
support `ros2 action type <action name>`.

### PR DESCRIPTION
**Pull request by fujitatomoya**
address https://github.com/ros2/ros2cli/issues/891

### CODE REVIEW

**Review by fujitatomoya — COMMENTED** [review]
@clalancette @audrow can you take a look at this? if we take this fix into Jazzy, i will add this to release note as well.

**Review by mjcarroll — APPROVED** [review]

File: ros2action/setup.py [hunk 1]
```diff
@@ -43,6 +43,7 @@
             # 'echo = ros2action.verb.echo:EchoVerb',
             'info = ros2action.verb.info:InfoVerb',
             'list = ros2action.verb.list:ListVerb',
+            'type = ros2action.verb.type:TypeVerb',
             'send_goal = ros2action.verb.send_goal:SendGoalVerb',
```
Comment (clalancette) [inline ros2action/setup.py:47]: Alphabetical order, please:

```suggestion
            'send_goal = ros2action.verb.send_goal:SendGoalVerb',
            'type = ros2action.verb.type:TypeVerb',
```

File: ros2action/setup.py [hunk 2]
```diff
@@ -43,6 +43,7 @@
             # 'echo = ros2action.verb.echo:EchoVerb',
```
Comment (clalancette) [inline ros2action/setup.py:43]: I know you didn't add this in, but while we are in here let's remove this comment.  We'll add it back in if/when we implement this.

**Review by clalancette — CHANGES_REQUESTED** [review]
I've left a couple of minor changes inline.

Besides that, for some reason I can't get this to work locally.  When I compile it and run it, the `type` verb doesn't show up for me.  Does it work for other people?

**Comment by mjcarroll** [comment]
> Does it work for other people?

Do you maybe need to source/re-source the workspace?

```
$ ros2 action
usage: ros2 action [-h] Call `ros2 action <command> -h` for more detailed usage. ...

Various action related sub-commands

options:
  -h, --help            show this help message and exit

Commands:
  info       Print information about an action
  list       Output a list of action names
  send_goal  Send an action goal
  type       Print a action's type

  Call `ros2 action <command> -h` for more detailed usage.
$ ros2 action type
usage: ros2 action type [-h] action_name
ros2 action type: error: the following arguments are required: action_name
```

**Comment by clalancette** [comment]
> Do you maybe need to source/re-source the workspace?

Yeah, it is working now.  I don't know why I could get it working yesterday.

File: ros2action/setup.py (hunk 1 above)
Comment (fujitatomoya) [reply ros2action/setup.py]: addressed by https://github.com/ros2/ros2cli/pull/894/commits/55f271b502d7cd83130201a807aa4c17d4847444

File: ros2action/setup.py (hunk 2 above)
Comment (fujitatomoya) [reply ros2action/setup.py]: addressed by https://github.com/ros2/ros2cli/pull/894/commits/55f271b502d7cd83130201a807aa4c17d4847444

**Comment by fujitatomoya** [comment]
CI:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/20842/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/15145/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/21528/)

**Comment by fujitatomoya** [comment]
note for myself: we need to add this to https://github.com/ros2/ros2_documentation/blob/rolling/source/Releases/Release-Jazzy-Jalisco.rst

**Review by clalancette — APPROVED** [review]
Looks good to me with green CI.

**Pull request merged**